src/js/components/DataSearch/DataSearch.js

```diff
@@ -77,6 +90,9 @@ export const DataSearch = ({ drop, id: idProp, responsive, ...rest }) => {
     />
   );
 
+  if (updateOn === 'submit')
+    content = <Keyboard onEnter={updateView}>{content}</Keyboard>;
```

Comment:
Is there anything from a screen reader standpoint that we should add to clarify how to submit the search? Or is hitting enter already an assumed way to submit? @jcfilben do you have insights from your work with Bill?

Reply:
I tested the SelectMultiple search with Bill and he mentioned that for a search input he always expects to be able to hit enter and then the screenreader should give some indication that something actually happened. So I think hitting enter is already the assumed way to submit.

Reply:
So for the default of `change` the search will happen before you'd hit enter (and hitting enter won't actually do anything since it's already done). But does that mean we need some announce context?

Reply:
Just tested with voiceover, when you hit enter (regardless of whether the search is trigger onChange or onSubmit) the screenreader will read out the summary text. I think this is a good screenreader experience but we can always validate with Bill. Either way I think we can move forward with this PR and if we need to make any screenreader adjustments we can do that separately.